articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+    eleventyConfig.addPassthroughCopy('img');
+    eleventyConfig.addPassthroughCopy('css');
+    eleventyConfig.addPassthroughCopy('js');
+    return {
+        passthroughFileCopy: true
+    };
+};
+```
+This will tell Eleventy when it builds your website to just copy the img, CSS and js folders across (which is called passthrough copy) so you can link them to them in your code.
+
+Now run `npx @11ty/eleventy` and it should have built your website and output the result to the `_site` folder. Congratulations, you've just finished the first Eleventy build of your website.
+
+**Tip:** If you're using the version control software, Git **link to a section article on Git** manages your code then it would be a good idea to add the `_site` folder to your .gitignore file. This will stop you from committing your `_site` folder to Git which prevents the constant uncommitted changes warning.
+
+### Creating Your First EJS Partial
+
+Now that your site is up and running on Eleventy, we can get to the templating part.
+
+EJS partials are a way of reusing pieces of code for example, if you created an EJS partial of your website's header then it could be referenced on all your website's pages. In the future, if you decided to add a new menu item then you would only need to update one file.
+
+Create a folder in the root of your website called `_includes`. This is where Eleventy looks for templates.
+
+**Tip:** A naming convention you may wish to use is to add an `_` to your template files so `_head.ejs`.
+
+In your `_includes` folder, add a file called `_head.ejs` and copy and paste the content of your website's head section. Don't worry if the content differs between pages such as the title tag. We will cover how to use EJS variables with Eleventy in the Adding More Front-Matter section **Anchor Link** so such values can be unique for each page.
+
+Add new partial files for other repeated elements such as the header and footer.
+
+### Creating Your First Layout (Template) File
+
+With the partials created, you can now create your first layout file (template.)
+
+The first layout (often named base layout) will form the basic template of all your pages and will contain the EJS partials (head, header and footer) that you've just created.
+
+**Tip:** Another naming convention that you may want to use is to add -layout at the end of your layout files. This helps to differentiate them from your partials.
+
+In the `_includes` folder, create a file called `_base-layout.ejs` file and add the following to it:
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <% include _head.ejs %>
+</head>
+<body>
+    <% include _header.ejs %>
+    <main>
+        <%- content %>
+    </main>
+    <% include _footer.ejs %>
+</body>
+</html>
+```
+If you're new to EJS, `<% include` is the code (syntax) for including a partial file and `<%-` is for including an EJS variable. Content is a variable Eleventy uses to add the body content of HTML, Markdown and other input files.
```

Comment:
This is great! You should find as many places as possible to explain your code this way. Simple, to the point, yet very informative. 

My only suggestion here is that you should remove the "code (syntax)" and replace it with just "syntax". If your reader does not know what syntax is then they are reading the wrong article.

Reply:
Thanks. I think it's good to provide both an example and an explanation of how it works.

Removed word as suggested in fe0a337